firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/ScreenshotTaker.java

```diff
@@ -40,21 +39,16 @@ class ScreenshotTaker {
 
   private final FirebaseApp firebaseApp;
   private final FirebaseAppDistributionLifecycleNotifier lifecycleNotifier;
-  private final Executor taskExecutor;
-
-  ScreenshotTaker(
-      FirebaseApp firebaseApp, FirebaseAppDistributionLifecycleNotifier lifecycleNotifier) {
-    this(firebaseApp, lifecycleNotifier, Executors.newSingleThreadExecutor());
-  }
+  private final Executor backgroundExecutor;
 
   @VisibleForTesting
```

Comment:
remove this annotation - this constructor is now called from `FirebaseAppDistributionRegistrar`

Reply:
Ah good catch